deploy/README.md

```diff
@@ -0,0 +1,309 @@
+    wikibase-suite_mysql-data \
+    wikibase-suite_wdqs-data \
+    wikibase-suite_elasticsearch-data \
+    wikibase-suite_quickstatements-data \
+    wikibase-suite_traefik-letsencrypt-data; do
+  docker volume rm $v 2> /dev/null
+  docker volume create $v
+  docker run -i --rm --volume $v:/backup debian:12-slim tar xz < $v.tar.gz
+done
+```
+
+### Updating
+
+WBS is versioned with [Semantic Versioning](https://semver.org/spec/v2.0.0.html). The WBS Deployment Kit and all the WBS Service Images have individual version numbers.
+
+> 💡 WBS Deployment Kit always references the latest minor and patch releases of the compatible WBS Service Images major versions.
+
+#### Minor and patch updates for WBS Service Images
+
+As WBS Deployment Kit always references latest minor and patch releases of compatible WBS Service Images TODO: rephrase, non breaking changes (including security updates) are applied automatically when recreating Docker containers. This should be always safe to do. Simply run
+
+```sh
+docker compose down 
+docker compose up --wait
+```
+
+> 💡 If you do not want to pull in new versions of WBS Service Images on container restart, just stop your containers using `docker compose stop` before restart. This will keep the current containers intact. Please note that this will not apply any security updates. It is generally recommended to remove your containers for restart with `docker compose down`.
+
+#### Minor and patch updates for WBS Deployment Kit
+
+WBS Deployment Kit major versions are tracked in dedicated branches such as `TODO:BRANCH_NAME_HERE`. Therefor, pulling from such a branch will only update minor and patch version but never trigger breaking changes.
+
+If you did not change `docker-compose.yml`, you can update simply using `git pull`.
+
+```sh
+# TODO check BRANCH/TAG before release
+git pull
+```
+If you have made complex changes to `docker-compose.yml`, `git commit` them to a separate branch and `git merge` them with upstream changes as you see fit.
+
+> 💡 One major version of WBS Deployment Kit will always reference the only one major version for each of the WBS Service Images. Therefor, updating WBS Deployment Kit minor and patch versions from its major versions git branch will never lead to breaking changes in WBS Service Images. Those updates are always considered safe!
+
+#### Major upgrades
+
+Major version upgrades are performed by updating WBS Deployment Kits major version. This might reference new major versions of WBS Service Images too and therefor include breaking changes. Those may require additional steps. Refer to specific upgrade instructions in the [Migration Guide](#migration-guide) section below.
+
+WBS only supports updating from one major version to the next one. If you want to upgrade from 21 to 23, you need to upgrade from 21 to 22 first and in a next step from 22 to 23.
+
+> ‼️ Always [create a backup](#backup-your-data) of your data before performing a major version update.
```

Comment:
"Also backup changes to `docker-compose.yml` and look for changes in the `template.env` file."

Reply:
Good point, leaving it open as a reminder to review the major update procedure.